apps/package.json

```diff
@@ -145,7 +145,6 @@
     "load-grunt-tasks": "3.5.0",
     "loadable-components": "2.2.3",
     "lodash": "^4.17.5",
-    "marked": "^0.3.6",
```

Comment:
I think you're missing a corresponding yarn.lock change.

Reply:
Good eye! Actually, though, `marked` is still used by some of our other dependencies, so it's not actually being removed from our build, just our codebase.

Reply:
[o_O] Oh dang! yarn.lock didn't change because `webpack-notifier@^1.4.1` depends on `node-notifier@^4.1.0` depends on `cli-usage@^0.1.1` depends on `marked` at this _exact_ semver expression.

![nothing-but-net](https://user-images.githubusercontent.com/1615761/79503865-bb4fdb80-7fe6-11ea-9e76-1b9eed4de1b5.gif)

Reply:
Haha, exactly!